src/rust/multi.comick/src/lib.rs

```diff
@@ -10,8 +10,8 @@ use aidoku::{
 pub mod helper;
 pub mod parser;
 
-const BASE_URL: &str = "https://comick.app";
-const API_URL: &str = "https://api.comick.app";
+const BASE_URL: &str = "https://comick.cc";
+const API_URL: &str = "https://api.comick.fun";
```

Comment:
```suggestion
const API_URL: &str = "https://api.comick.cc";
```

Reply:
The API Docs specifically mention to use `https://api.comick.fun`.

https://api.comick.fun/docs/static/index.html
![image](https://github.com/Skittyblock/aidoku-community-sources/assets/28512568/4482f5a4-9403-4010-9590-b9a9eb3405af)

Should this still be .cc or do we respect the docs?